optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/uni/UniConstraintStreamTest.java

```diff
@@ -1419,45 +1419,6 @@ public void duplicateConstraintId() {
                 constraintStreamImplType));
     }
 
-    @TestTemplate
-    public void globalNodeOrder() {
```

Comment:
Not useful anymore. We now have a better test, and that one checks the node indexes directly.

Reply:
By removing it, this test also doesn't run on CS-D any more?
+0 to remove if it's fully duplicate for Bavet, including the actual scoreCalculation verification.
Your call.

If we do keep it, maybe it should be renamed to something like "nodeIndexChallenger" or "joinFilterLeftBridgeVersusJoinFilterRightBridge"? Idunno.

Reply:
CS-D never had node indexes, we need to worry about these things there.
Everything else this test does is covered elsewhere.